src/web/mjs/connectors/ReadMNG.mjs

```diff
@@ -71,13 +71,21 @@ export default class ReadMNG extends Connector {
             } );
     }
 
+    /*
+    * Used to check if the img is not an ad
+    */
+    checkMangaImage(img) {
+        return img.classList.contains('img-responsive');
+    }
+
     /**
      *
      */
     _getPageList( manga, chapter, callback ) {
         let request = new Request( this.url + chapter.id + '/all-pages', this.requestOptions );
         this.fetchDOM( request, 'div.page_chapter source' )
```

Comment:
Could this filtering be done in the CSS query, with `:not(..)` or match operator?
something like `div.page_chapter source[class*="img-responsive"]`

Reply:
Indeed, your filter worked nicely. I have not done such filtering for years, thanks for the help.

Tested again (preview/download): ok